gitpod-io/gitpod
# Refactor installer to match team structure

## Description
This PR refactors the structure of the installer so that the teams gain more control over their part of the installation. This will be important when introducing component-specific unsupported configuration.

## Related Issue(s)

part of #7776

## How to test
Compare the output of the installer before and after this change.
Note: prior to this build the order of the generated objects was determined by the order of the objects. When testing with an "old" installer, make sure to include the sorting change.

## Release Notes

```release-note
NONE
```
